TypeCobol.Analysis.Test/BasicCfgInstrs/ExtendedPerformProc1Recursive0.dot

```diff
@@ -0,0 +1,64 @@
+﻿digraph Cfg {
+compound=true;
+node [
+shape = "record"
+]
+
+edge [
+arrowtail = "empty"
+]
+Block0 [
+label = "{START|}"
+]
+Block1 [
+label = "{Block1|    perform pararec\l}"
+]
+Block10 [
+label = "{PARAREC. Block10|    add 1 to n\l    display n\l    if n \< 3\l}"
+]
+Block11 [
+label = "{Block11|       display 'before perform'\l}"
+]
+Block12 [
+label = "{Block12|       perform pararec\l}"
+]
+Block21 [
+label = "{PARAREC. Block21|    add 1 to n\l    display n\l    if n \< 3\l}"
+]
+Block22 [
+label = "{Block22|       display 'before perform'\l}"
+]
+Block24 [
+label = "{Block24|    else\l       display 'not done'\l}"
+]
+Block25 [
+label = "{Block25|    display 'terminate'\l}"
+]
+Block13 [
+label = "{Block13|       display 'after perform'\l}"
+]
+Block15 [
+label = "{Block15|    display 'terminate'\l}"
+]
+Block2 [
+label = "{Block2|    goback\l}"
+]
+Block14 [
+label = "{Block14|    else\l       display 'not done'\l}"
+]
+Block0 -> Block1
+Block1 -> Block10
+Block10 -> Block11
+Block10 -> Block14
+Block11 -> Block12
+Block12 -> Block21
+Block21 -> Block22
+Block21 -> Block24
+Block22 -> Block12
+Block24 -> Block25
+Block25 -> Block13
+Block13 -> Block15
+Block15 -> Block2
+Block14 -> Block15
+
+}
```

Comment:
`ExtendedPerformProc3Recursive0.dot` is not present here, but after checking it seems that the file is missing the block instruction "after perform pararec in pararec2" (if it is the extended version of `PerformProc3Recursive0.dot`)

Reply:
As discussed, a control-flow graph (CFG) is a representation, using graph notation, of all paths that might be traversed through a program during its execution. So in that case the Extended version is compliant with the given definition.